### ISSUE TITLE
Storage options for physical data

### ISSUE THREAD

**Issue by Alex-van-der-Jagt** [ubvu/open-handbook#144]
Currently there are a fair few descriptions on storage for digital storage, but it looks to me that storage for phsycial data (such as saliva samples, hair samples, but also paper questionnaires) is glossed over.

**Comment by chartgerink** [comment]
Would you consider this to be a topic called "Physical data storage"?

**Comment by Alex-van-der-Jagt** [comment]
Yes, that would be a good section. It can be added to the Collect and Store data pathway

**Comment by jhrudey** [comment]
Hey @Alex-van-der-Jagt does this FGB page cover what you are looking for https://fgb-rdm.nl/Security/PhysicalTransport.html or do you want something with even more detail? The one I made was primarily so people would stop losing their unencrypted laptops, so it doesn't get into all the detail you mention. 
In either case, I can get a draft started and we can look at what we want this section to be (and it can include a cross reference to the physical transport page I've already made)

**Comment by jhrudey** [comment]
P.S. before I make a draft do the folks here think we need a separate topic page on Physical Data Storage or could it be a subsection on this page https://rdm.vu.nl/topics/data-storage.html
